addons/mail/static/src/messaging/entity/device/device.js

```diff
@@ -7,8 +7,28 @@ function DeviceFactory({ Entity }) {
 
     class Device extends Entity {
 
+        /**
+         * @override
+         */
+        static create() {
+            const entity = super.create();
+            entity.start();
```

Comment:
I would delegate the start to messagingInitializer as it is also taking care of the stop.